modules/lymphgen/1.0/lymphgen.smk

```diff
@@ -409,10 +550,6 @@ rule _lymphgen_output_txt:
 
 # Set the applicable wildcards, based on the provided input files
 # Are we running LymphGen with CNV/SV data?
-if "sample_seg" in CFG["inputs"] and CFG["inputs"]["sample_seg"] != "" and CFG["inputs"]["sample_seg"] != "None":
-    cnvs_wc = ["with_cnvs", "no_cnvs"]
-else:
-    cnvs_wc = ["no_cnvs"]
 
 if "sample_sv_info" in CFG["inputs"] and CFG["inputs"]["sample_sv_info"] != "" and CFG["inputs"]["sample_sv_info"] != "None":
```

Comment:
Is there more to do here to make sure this works on a per-sample basis? The way you have it set up, the `sample_sv_info` field in the config will never be "None", so this might need more refinement. Ditto for A53, which should maybe be set at the same time as the CNV info is set (e.g. the RUNS_NO_CNV table should only be expanded on `no_cnv` and `no_A53` and RUNS_CNV should be expanded on `["with_cnv", "no_cnv"]` and `["with_A53", "no_A53"]`?